### PR TITLE
adds expansion content to Web Lit Basics module

### LINKED ISSUE #2448: Add privacy activities to the Web Lit Basics I module
@hannahkane, we'd like to add these activities to the Web Lit Basics I module to make sure that collection covers the Web Literacy skill "Protect."

Page content update here: https://docs.google.com/document/d/1aOlD4RUqmr2pu0EUMN4G8xXJWE1naFimu3MQigbwFvY/edit#heading=h.ekhodybunbfe

We do not want to do anything to the Privacy Basics: Protect Your Data module. The 3 activities we want to add will appear in both modules.

cc @anmechung @ibgmozilla

### PR DESCRIPTION
**Pull request by hannahkane**
- [x] Localization addressed - NA

Fixes #2448

**Changes proposed in this pull request:**
- adds "Privacy Expansion" section to Web Lit Basics module

### CODE REVIEW

**Comment by hannahkane** [comment]
@flukeout - do you have a moment to look at this, please?

File: pages/web-lit-basics.jsx [hunk 1]
```diff
@@ -68,6 +68,34 @@ var curriculumList = [
         link: "https://mozilla.github.io/curriculum-final/web-lit-basics-one/session06-story-of-us.html#overview"
       }
     ]
+  },
+  {
+    title: "Privacy Expansion",
+    activities: [
+      {
+        title: "Create Secure Passwords",
+        image1x: "/img/pages/protect-your-data/pass-phrase.jpg",
```
Comment (mmmavis) [inline pages/web-lit-basics.jsx]: Let's make sure CC license info is included here as well.

```jsx
{
  ...
  originalImgSrc: "https://mozilla.github.io/curriculum-final/privacy-basics/images/pass-phrase-1.jpg",
  caption: "CC-BY Aurich Lawson",
  ...
}
```

File: pages/web-lit-basics.jsx [hunk 2]
```diff
@@ -68,6 +68,34 @@ var curriculumList = [
...
+        subtitle: "Evaluate, Protect",
+        description: "You will learn several different ways to generate secure passwords and pass-phrases and test their strength, learning web literacy skills like <strong>evaluate</strong> and <strong>protect</strong>.",
+        link: "https://mozilla.github.io/curriculum-final/privacy-basics/session04-create-secure-passwords.html"
+      },
+      {
+        title: "Cookies and Third-Party Tracking",
+        image1x: "/img/pages/protect-your-data/img-cookies.png",
+        image2x: "/img/pages/protect-your-data/img-cookies@2x.png",
```
Comment (mmmavis) [inline pages/web-lit-basics.jsx]: Let's add CC license info here.

File: pages/web-lit-basics.jsx [hunk 3]
```diff
@@ -68,6 +68,34 @@ var curriculumList = [
...
+        subtitle: "Navigate, Open Practice, Protect",
+        description: "You will use a privacy tool called Mozilla Lightbeam to see how websites track your movement online, learning web literacy skills like <strong>navigate</strong>, <strong>open practice</strong>, and <strong>protect</strong>.",
+        link: "https://mozilla.github.io/curriculum-final/privacy-basics/session03-cookies-and-third-party-tracking.html"
+      },
+      {
+        title: "Data Trail Timeline",
+        image1x: "/img/pages/protect-your-data/img-datatrail.png",
+        image2x: "/img/pages/protect-your-data/img-datatrail@2x.png",
```
Comment (mmmavis) [inline pages/web-lit-basics.jsx]: Let's add CC license info here.

File: pages/web-lit-basics.jsx [hunk 4]
```diff
@@ -68,6 +68,34 @@ var curriculumList = [
...
+  },
+  {
+    title: "Privacy Expansion",
+    activities: [
+      {
+        title: "Create Secure Passwords",
+        image1x: "/img/pages/protect-your-data/pass-phrase.jpg",
+        subtitle: "Evaluate, Protect",
+        description: "You will learn several different ways to generate secure passwords and pass-phrases and test their strength, learning web literacy skills like <strong>evaluate</strong> and <strong>protect</strong>.",
+        link: "https://mozilla.github.io/curriculum-final/privacy-basics/session04-create-secure-passwords.html"
```
Comment (mmmavis) [inline pages/web-lit-basics.jsx]: link should be https://mozilla.github.io/curriculum-final/web-lit-basics-one/session07-create-secure-passwords.html#overview ?

File: pages/web-lit-basics.jsx [hunk 5]
```diff
@@ -68,6 +68,34 @@ var curriculumList = [
...
+        description: "You will use a privacy tool called Mozilla Lightbeam to see how websites track your movement online, learning web literacy skills like <strong>navigate</strong>, <strong>open practice</strong>, and <strong>protect</strong>.",
+        link: "https://mozilla.github.io/curriculum-final/privacy-basics/session03-cookies-and-third-party-tracking.html"
+      },
+      {
+        title: "Data Trail Timeline",
+        image1x: "/img/pages/protect-your-data/img-datatrail.png",
+        image2x: "/img/pages/protect-your-data/img-datatrail@2x.png",
+        subtitle: "Protect",
+        description: "You will create a timeline on a poster or in a video or slideshow to demonstrate how information about your movement in the real world gets collected online by companies and other organizations throughout the course of a typical day while learning about web literacy skills like <strong>protect</strong>.",
+        link: "https://mozilla.github.io/curriculum-final/privacy-basics/session05-data-trail-timeline.html"
```
Comment (mmmavis) [inline pages/web-lit-basics.jsx]: https://mozilla.github.io/curriculum-final/web-lit-basics-one/session09-data-trail-timeline.html#overview  ?

File: pages/web-lit-basics.jsx [hunk 6]
```diff
@@ -68,6 +68,34 @@ var curriculumList = [
...
+        description: "You will learn several different ways to generate secure passwords and pass-phrases and test their strength, learning web literacy skills like <strong>evaluate</strong> and <strong>protect</strong>.",
+        link: "https://mozilla.github.io/curriculum-final/privacy-basics/session04-create-secure-passwords.html"
+      },
+      {
+        title: "Cookies and Third-Party Tracking",
+        image1x: "/img/pages/protect-your-data/img-cookies.png",
+        image2x: "/img/pages/protect-your-data/img-cookies@2x.png",
+        subtitle: "Navigate, Open Practice, Protect",
+        description: "You will use a privacy tool called Mozilla Lightbeam to see how websites track your movement online, learning web literacy skills like <strong>navigate</strong>, <strong>open practice</strong>, and <strong>protect</strong>.",
+        link: "https://mozilla.github.io/curriculum-final/privacy-basics/session03-cookies-and-third-party-tracking.html"
```
Comment (mmmavis) [inline pages/web-lit-basics.jsx]: https://mozilla.github.io/curriculum-final/web-lit-basics-one/session08-cookies-and-third-party-tracking.html#overview  ?

**Comment by hannahkane** [comment]
@mmmavis - how's that? (btw, I didn't mean to take up so much of your time. This is likely not more urgent than what you're working on.)

**Comment by mmmavis** [comment]
@hannahkane no worries! it's a nice short break for me

**Pull request merged**